lib/blobStorage.ts

```diff
@@ -237,6 +237,16 @@ export class BlobStoragePrefix {
   }
 }
 
+/**
+ * Options that can be applied when updating a blob's contents.
+ */
+export interface BlobContentOptions {
+  /**
+   * The MIME content type that will be associated with this blob's content.
+   */
+  contentType?: string;
```

Comment:
Do we know set of content types it supports? If so, we can limit it here.

Reply:
It just takes content-type as a string, so it can support anything (I use "abc" in my tests).